tests/performance/ci_run.sh

```diff
@@ -45,7 +45,7 @@ MARATHON_VERSION=$("$MARATHON_DIR/version" docker)
 DOCKER_COMPOSE_FILE="$BASEDIR/config/docker-compose.yml"
 
 # Start docker-compose, detached
-export MESOS_VERSION=1.5.1-rc1
+export MESOS_VERSION=1.7.0
```

Comment:
Shouldn't we also bump the `Dependencies.scala` (and check that it works) then?

Reply:
That's actually a good point 👍

Reply:
@timcharper I talked with the mesos guys regarding our performance tests and they mentioned that we are using an outdated mesos version. And as a matter of fact, they have changed quite a few things on 1.7. 

So I thought that since we are integrating with the latest version of mesos, to actually run the performance tests against the latest mesos channel.

Reply:
Ouch, now that I think of it, I also need to have to make a new build for the mock-executor in order to be compatible with the 1.7 API.

Reply:
Are we going to bump this or close it then?